### DISCUSSION TITLE
Уведомление при обновлении Goofy

### DISCUSSION THREAD

**Discussion by amd64fox** [Chimildic/goofy#50] [Избранное]
Где-то недавно попадалось на глаза фишка про уведомление, если goofy обновился, хотел узнать почему она пропала ?  По мне так идея хорошая, если бы например уведомление приходило на почту, ну или вариант чуть по сложнее, через бота в телеге.

**Comment by Chimildic** [comment]
Можно получить уведомления внешними средствами. Если дать им следующую ссылку, будут приходить сообщения при появлении новых коммитов (обновлении кода): `https://github.com/Chimildic/goofy/commits/main.atom`
- В телеграм через бота [rss2tg](https://t.me/rss2tg_bot).
- Через почту [здесь](https://www.feed2mail.com/). Тоже есть бот для телеграма. 
- Расширение для браузера [Feedbro](https://nodetics.com/feedbro/). Для уведомлений нужно добавить правило `rule`.
- Или в любой альтернативе отслеживания `RSS/atom` под другие платформы/места.
